js/huobi.js

```diff
@@ -3088,4 +3090,72 @@ module.exports = class huobi extends Exchange {
         const sorted = this.sortBy (rates, 'timestamp');
         return this.filterBySymbolSinceLimit (sorted, symbol, since, limit);
     }
+
+    parseFundingRate (fundingRate, market = undefined) {
+        //
+        // {
+        //      "status": "ok",
+        //      "data": {
+        //         "estimated_rate": "0.000100000000000000",
+        //         "funding_rate": "0.000100000000000000",
+        //         "contract_code": "BCH-USD",
+        //         "symbol": "BCH",
+        //         "fee_asset": "BCH",
+        //         "funding_time": "1639094400000",
+        //         "next_funding_time": "1639123200000"
+        //     },
+        //     "ts": 1639085854775
+        // }
+        //
+        const previousFundingRate = this.safeNumber (fundingRate, 'funding_rate');
+        const nextFundingRate = this.safeNumber (fundingRate, 'estimated_rate');
+        const previousFundingTimestamp = this.safeInteger (fundingRate, 'funding_time');
+        const nextFundingTimestamp = this.safeInteger (fundingRate, 'next_funding_time');
+        return {
+            'info': fundingRate,
+            'symbol': market['symbol'],
```

Comment:
Market may be undefined, so you need to use `this.safeSymbol (market, 'symbol'), because if market isn't defined, you get an error when you do this

```
$ node
> market['symbol']
Uncaught ReferenceError: market is not defined
```